### PR TITLE
Updated references.rst

### PR DESCRIPTION
**Pull request by sonikarajesh**
Added links to Tutorials (radis/spectro101)

### CODE REVIEW

File: docs/references/references.rst [hunk 1]
```diff
@@ -60,6 +60,13 @@ Research papers using RADIS and the associated algorithms :
 
 - Papers citing |badge_article2| : https://scholar.google.fr/scholar?cites=17363432006874800849
 
+Tutorials
+=============
```
Comment (erwanp) [inline docs/references/references.rst:64]:
```suggestion
Spectroscopy Tutorials
=====================
```

**Review by erwanp — CHANGES_REQUESTED** [review]

File: docs/references/references.rst (hunk 1 above)
Comment (erwanp) [reply docs/references/references.rst:64]: @sonikarajesh you can commit directly from the web interface

**Comment by erwanp** [comment]
Btw, RADIS enforces code linting ; here the test failed (see Checks)

Help on how to lint : https://radis.readthedocs.io/en/latest/dev/developer.html#code-linting

**Comment by erwanp** [comment]
Can you check the linting errors?

**Comment by erwanp** [comment]
@sonikarajesh gentle reminder as we need to fix linting errors to merge

**Comment by sonikarajesh** [comment]
> @sonikarajesh gentle reminder as we need to fix linting errors to merge 
could you try merging?

**Pull request merged**